gax-java/gax/src/test/java/com/google/api/gax/rpc/internal/QuotaProjectIdHidingCredentialsTest.java

```diff
@@ -65,9 +62,9 @@ public void quotaProjectIdHidingCredentials_getRequestMetadata() throws IOExcept
     Map<String, List<String>> metaDataHidingQuota =
         quotaProjectIdHidingCredentials.getRequestMetadata();
 
-    Assert.assertTrue(metaDataWithQuota.containsKey(QUOTA_PROJECT_ID_KEY));
-    Assert.assertFalse(metaDataHidingQuota.containsKey(QUOTA_PROJECT_ID_KEY));
-    Assert.assertEquals(metaDataWithQuota.size() - 1, metaDataHidingQuota.size());
+    Assertions.assertTrue(metaDataWithQuota.containsKey(QUOTA_PROJECT_ID_KEY));
```

Comment:
nit: use static import for `Assertions`.

Reply:
Updated.